voters-guide-2017.md

```diff
@@ -0,0 +1,69 @@
+*  Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Kubernetes
+*  Charter and refine policy for defining new community groups (including Special Interest Groups, Working Groups, and Committees), and establish transparency and accountability policies for such groups.  
+*  Define, evolve, and defend a Code of Conduct, which must include a neutral, unbiased process for resolving conflict 
+
+Full scope of responsibilities, goals, and/or future election timelines, see [steering committee charter](https://github.com/kubernetes/steering/blob/master/charter.md). 
+
+For more context, please see the [current issues](https://github.com/kubernetes/steering/blob/master/backlog.md) that need to be resolved or a previous governance [meeting video](https://www.youtube.com/watch?v=ltRKXLl0RaE&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ&index=23) which lead to this whole process.
+
+## PROCESS
+Elections will be held using time-limited [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on [CIVS](http://civs.cs.cornell.edu/) using the [Schulze method](https://en.wikipedia.org/wiki/Schulze_method). The top vote getters, modulo the corporate diversity requirement of no more than 1/3 of the committee from a single company, will be elected to the respective positions, with the top 3 filling the 2 year term seats, and the next 3 filling the 1 year term seats. 
+
+The election will open for voting on September 19, 2017 at 09:00am PDT and end two weeks after on October 3, 2017 at 06:00pm PDT. You will receive an email to the address on file at the start of the election from <civs@cs.cornell.edu>; please whitelist if necessary. 
+
+## ELIGIBILITY 
+Members of Standing are defined by the union of:
+SIG leads
+Approvers and reviewers in any Kubernetes owned repositories
+Anyone with write access to a Kubernetes owned repository
+Active members of our community
+If you believe you are a Member in Standing, please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSeoeNSl9ufZ_jpp7OHgvtWm-GRFV6WUwTIqZ9W25eMd3xyyvg/viewform) <strong><em>before</strong></em> September 13, 2017.	
+
+## DECISION
+The newly elected body will be announced in the weekly Kubernetes Community Meeting on October 5, 2017 at 10:00am US Pacific Time. [Please join us](https://groups.google.com/forum/#!forum/kubernetes-community-video-chat).
+
+Immediately following the meeting, the raw voting results and winners will be published on the [Kubernetes Blog](http://blog.kubernetes.io/).
+
+For more information, definitions, and/or detailed election process, see full [steering committee charter](https://github.com/kubernetes/steering/blob/master/charter.md).
+
+# NOMINEES
+Don’t know someone on this list? Check out their K8s bio by first+last name [here](https://github.com/kubernetes/community/tree/master/community/elections/2017). 
+
```

Comment:
Maybe we can add a column to link to every nominees' bio link?

Reply:
will add the link to the bios in the name section as they come in